templates/event/show.html.twig

```diff
@@ -8,7 +8,9 @@
             <h1 class="event_show_page_title">{{ event.name }}</h1>
         </div>
         <div class="event_show_content d-flex">
-            <img class="col-12 col-lg-4" src="{{ asset (event.poster) }}" />
+        {% if event.poster.slug is defined %}
```

Comment:
You probably should test if it's not empty instead

Reply:
Ok 'is not empty' would be more effective...